frontend/src/core/utils/fluent/types.js

```diff
@@ -1,56 +1,12 @@
 /* @flow */
 
-export type Identifier = {
-    type: string,
-    name: string,
-};
+import type { Attribute, Entry } from '@fluent/syntax';
 
-export type NumberLiteral = {
-    type: string,
-    value: string,
-};
-
-export type Variant = {
-    default: boolean,
-    key: NumberLiteral | Identifier,
-    value: Pattern,
-};
-
-export type SelectExpression = {
-    type: string,
-    variants: Array<Variant>,
-};
-
-export type Placeable = {
-    type: string,
-    expression: SelectExpression,
-};
-
-export type PatternElement =
-    | {|
-          type: string,
-          value: string,
-      |}
-    | Placeable;
-
-export type Pattern = {
-    elements: Array<PatternElement>,
-};
-
-export type FluentAttribute = {
-    id: { name: string },
-    value: Pattern,
-};
+export type FluentAttribute = Attribute;
 
 export type FluentAttributes = Array<FluentAttribute>;
 
-export type FluentMessage = {
-    type: string,
-    value: Pattern,
-    attributes: ?FluentAttributes,
-    clone: () => FluentMessage,
-    equals: (any) => boolean,
-};
+export type FluentMessage = Entry;
```

Comment:
I'm torn on whether to keep these type aliases or not.

In particular the mismatch between calling this a message when it actually is a term, or even a Junk entry, bothers me.

Reply:
What would be the value of keeping these aliases vs just using the types directly from `@fluent/syntax`

Reply:
TBH, I've not asked anybody when they got introduced. I'm not even sure if that was TN or translate.old 🤷 

My guess is that it's namespaced as Fluent-specific, but I haven't found that to be much of a problem in the code I touched, it was usually very straight-forward that it's code handling Fluent.

Reply:
We talked about this, and we should drop those aliases